apk/Dockerfile-3.12

```
@@ -0,0 +1,11 @@
+FROM alpine:3.11
```

```suggestion
FROM alpine:3.12
```

i's `Dokerfile-3.12`, but `alpine:3.11`